From 8497bd742e8145d7ea67bb51f9914f28d5e18967 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 18 Dec 2019 15:05:28 +0800
Subject: [PATCH] ARM: rockchip_defconfig: disable CONFIG_I2C_HID

No android board use hid-over-i2c.
before:
   text         data        bss     dec             hex filename
12536545	4902620	2158240	19597405	12b085d	vmlinux
after:
   text            data     bss     dec             hex filename
12530111	4902420	2158240	19590771	12aee73	vmlinux

Save about 6K.

Change-Id: Idc64bafe267d614041faf7ac0355fffabba8fc36
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index cefd5b740acb..5cd0772b69fe 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -538,7 +538,6 @@ CONFIG_SND_SIMPLE_CARD=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=y
 CONFIG_USB_HIDDEV=y
-CONFIG_I2C_HID=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_MON=y
 CONFIG_USB_EHCI_HCD=y
-- 
2.35.3

